tests/modules/images/regenerate-existing-images/background-process/perflab-background-job-test.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * Tests for background-job module.
+ *
+ * @package performance-lab
+ * @group   regenerate-existing-images
+ */
+
+/**
+ * Class Perflab_Background_Job_Test
+ *
+ * @coversDefaultClass Perflab_Background_Job
+ * @group regenerate-existing-images
+ */
+class Perflab_Background_Job_Test extends WP_UnitTestCase {
+
+	/**
+	 * Job instance.
+	 *
+	 * @var Perflab_Background_Job
+	 */
+	private $job;
+
+	public function set_up() {
+		$this->job = perflab_create_background_job( 'test' );
+	}
```

Comment:
There are two issues here:
* You need to also call the parent method, otherwise this code causes problems.
* You also need to implement a `tear_down()` method that always deletes the job again, otherwise the database gets polluted for other tests.
```suggestion
	public function set_up() {
		parent::set_up();
		$this->job = perflab_create_background_job( 'test' );
	}

	public function tear_down() {
		perflab_delete_background_job( $this->job->get_id() );
		parent::tear_down();
	}
```